script/QA/QA_18.txt

```diff
@@ -116,19 +116,20 @@ We do not have much time, so perhaps you%{n}
 [sha:6eacf6c5abfd1f9a2a42a0b76dfc16668b9ade4d]{
 -- Page 122, Offset 43542.
 -- 今回の死因について解説をお願いします。
-could begin explaining what caused this death.
+could begin by explaining what caused this%{n}
+death.
 }
 [sha:85d876049a4994a0f858759edc173bae9d6d56c9]{
 -- Page 123, Offset 43543.
 -- うーん、下手に茶々いれると
 // Assuming this is an 'if we touch we'll annihilate' rules of the world thing
 // in the same vein as above
-Hmm, but I get the sense that if I mess%{n}
+Hmm, but I get the sense that if I'm not%{n}
 }
 [sha:ccde253fa43353b0a8077f46c90a78a3162015f3]{
 -- Page 123, Offset 43544.
 -- この世から消滅しそうなエクサトン。
-with her, an exaton of matter will be%{n}
+careful with her, an exaton of matter will be%{n}
 annihilated...
```

Comment:
Doesn't sound exactly right to me - the matter either annihilates something or is annihilated by something, and the change kinda makes it sound more like it happens in a vacuum, rather than it being annihilated _by neco arc_ (dark matter)

Reply:
I thought the anti-matter type of annihilation was intransitive

Reply:
Maybe? It just feels like an awkward construction, since the usual usage of annihilate _is_ transitive, and I worry it'll get misread. Maybe reword it? 
> ...not careful with her, it'll result in the annihilation of an exaton of matter.